test/SaveHandler/MongoDBTest.php

```diff
@@ -43,8 +44,10 @@ class MongoDBTest extends \PHPUnit_Framework_TestCase
      */
     public function setUp()
     {
-        if (!extension_loaded('mongo')) {
-            $this->markTestSkipped('Zend\Session\SaveHandler\MongoDB tests are not enabled due to missing Mongo extension');
+        if (!extension_loaded('mongodb')) {
```

Comment:
I think we should go rid of this check and use `@requires` annotations  https://phpunit.de/manual/current/en/incomplete-and-skipped-tests.html#incomplete-and-skipped-tests.requires.tables.api